build.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env -S node --import tsx/esm
```

Comment:
What is this exactly?

Reply:
It lets us write esbuild plugins in Typescript by enabling `node` to `import` `.ts` files.

Reply:
Grrmpfft why is that not the default.